com.arc.embeddedcdt/src/com/arc/embeddedcdt/gui/DebuggerGroupContainer.java

```diff
@@ -195,7 +195,9 @@ public void setFtdiDevice(FtdiDevice device){
     ftdiDevice = device;
   }
 
-  public String getOpenOcdConfigurationPath(){
+  /* This get-method is called with the word "access" because method
+   * "getOpenOcdConfigurationPath()" already exists. */
```

Comment:
It would be nice if you'd elaborate on why do you need to methods. I also think that you need to change their names to have bigger difference, so it would be easier to understand that one simply returns field value, while another does more complex evaluation.